SecurityException - App crashes when trying to add the "New Note" widget on home screen

- I have searched for existing issues that may be the same as or related to mine.

**Summary:**

The app crashes when adding the widget **New Note** to the home screen.

**Steps to Reproduce:**
1. Open your home screen (launcher)
2. Tap and hold on the home screen
3. Tap on **add widget**
4. Select **New Note** widget and drag it to your home screen

**Stack Trace:**

`Caused by: java.lang.SecurityException: Permission Denial: starting Intent { act=android.intent.action.CREATE_SHORTCUT cmp=com.orgzly/.android.ui.TemplateChooserActivity } from ProcessRecord{2ed9b75 7069:com.google.android.apps.nexuslauncher/u0a109} (pid=7069, uid=10109) not exported from uid 10151
        at android.os.Parcel.createException(Parcel.java:2071)
        at android.os.Parcel.readException(Parcel.java:2039)
        at android.os.Parcel.readException(Parcel.java:1987)
        at android.app.IActivityTaskManager$Stub$Proxy.startActivity(IActivityTaskManager.java:3851)
        at android.app.Instrumentation.execStartActivity(Instrumentation.java:1705)
        at android.app.Activity.startActivityForResult(Activity.java:5192)
        at com.android.launcher3.proxy.ProxyActivityStarter.onCreate(SourceFile:8)
        at android.app.Activity.performCreate(Activity.java:7802)
        at android.app.Activity.performCreate(Activity.java:7791)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1299)`

https://user-images.githubusercontent.com/54218218/183915060-a2561997-a2fb-48cb-b97a-a8ae8833bf3e.mp4

--- a/app/src/main/AndroidManifest.xml
+++ b/app/src/main/AndroidManifest.xml
@@ -150,7 +150,7 @@
             android:name=".android.ui.TemplateChooserActivity"
             android:label="@string/new_note_shortcut_label"
             android:icon="@mipmap/cic_shortcut_new_note"
-            android:exported="false">
+            android:exported="true">
 
             <intent-filter>
                 <action android:name="android.intent.action.CREATE_SHORTCUT" />

